benchmarks/density_ramp.py

```
@@ -0,0 +1,432 @@
+            mem_size_mib=tier_config["mem_size_mib"],
+            kernel_path=kernel,
+            rootfs_path=rootfs,
+            boot_args=SSH_BOOT_ARGS,
+            disk_mode=disk_mode,
+            backend=backend,
+        )
+        vm = SmolVM(config, ssh_key_path=ssh_key_path, socket_dir=socket_dir)
+        vm.start()
+        vm.wait_for_ssh(timeout=30.0)
+        return vm, time.time() - start_t, {}
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)